### DISCUSSION TITLE
Text sizing

### DISCUSSION THREAD

**Discussion by Jason-The-Shepherd** [SauceLLC/sauce4zwift#37] [General]
Is it possible to resize the text? I would like to be able to do this, especially for the chat. The size of the default Zwift chat text is about right for me. The Sauce chat widget is fantastic, but as a red beacon the text is a bit too small to read at a glance when monitoring so many other things and herding other riders .

**Comment by mayfield** [comment]
Hi @Jason-The-Shepherd 

You can adjust the zoom factor with Ctrl+Plus and Ctrl+Minus in the current release.  Each window type will remember it's zoom factor separately too.
